modules/nf-core/gecco/run/tests/main.nf.test

```diff
@@ -0,0 +1,68 @@
+nextflow_process {
+
+    name "Test Process GECCO_RUN"
+    script "../main.nf"
+    process "GECCO_RUN"
+
+    tag "modules"
+    tag "modules_nfcore"
+    tag "gecco"
+    tag "gecco/run"
+
+    test("gecco/run - candidatus_portiera_aleyrodidarum - genome_fasta") {
+
+        when {
+            process {
+                """
+                input[0] = [
+                    [ id:'test', single_end:false ], // meta map
+                    file(params.modules_testdata_base_path + 'genomics/prokaryotes/candidatus_portiera_aleyrodidarum/genome/genome.fasta', checkIfExists: true),
+                    []
+                ]
+                input[1] = []
+                """
+            }
+        }
+
+        then {
+            assertAll(
+                { assert process.success },
+                { assert snapshot(
+                    process.out.genes +
+                    process.out.features +
+                    process.out.clusters +
```

Comment:
Whats with the `+`? I've not seen that before?

Reply:
It's supposed to chain those outputs together and works. But thanks for the hint, I forgot to add the `.match()` which caused the test not to show up in the snapshot. Now it's fixed.

Reply:
And the `+` is used by a few modules already, e.g. multiqc.

Reply:
Ok... I wonder what it doss